src/promptflow-devkit/promptflow/_cli/_pf/_upgrade.py

```diff
@@ -109,6 +110,7 @@ def upgrade_version(args):
     importlib.reload(json)
 
     version_result = subprocess.check_output(["pf", "version"], shell=platform.system() == "Windows")
+    version_result = version_result.decode().replace("\x1b[0m", "").strip()
```

Comment:
add comment on what's this for?
\x1b[0m

Reply:
You cannot control this in multiple platforms, remove the color is wiser.

Or else, you can find the run_coverage_test.py and see how to print colored outputs.

Reply:
"\x1b[0m" is not added by sdk code, but added by termianl